src/components/SingleReview.jsx

```diff
@@ -41,12 +45,24 @@ export function SingleReview() {
         <p>{review.review_body}</p>
       </article>
       <p>Votes: {review.votes}</p>
+      <VoteButton reviewID={review.review_id} setReview={setReview} setDisplayAlert={setDisplayAlert} />
+      {displayAlert && <Alert crud='Vote' setDisplayAlert={setDisplayAlert} />}
```

Comment:
See my comment on the `Alert` component regarding all the elements to do with that.